assets/terraform-modules/bare-metal/flatcar-linux/kubernetes/ssh.tf

```diff
@@ -74,7 +77,7 @@ resource "null_resource" "copy-controller-secrets" {
       "sudo mv etcd-peer.key /etc/ssl/etcd/etcd/peer.key",
```

Comment:
We don't need to remove them I think, we can just copy existing key to `.old` or something. Rollback in case of failure I'd make manual (only one replica should fail anyway).

Reply:
added a backup